rsync/entrypoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Auth
+echo "$INPUT_CLOUD_CREDENTIALS" > /secrets.json
+gcloud auth activate-service-account --key-file=/secrets.json
+rm /secrets.json
+
+# Check destination
+if [[ $INPUT_TO =~ $INPUT_CHECK_DESTINATION ]]; then
+    echo "Destination exists"
+else
+    echo "Error - Destination doesn't exist"
+    exit 1
+fi
+
+# Validate the directory is not empty
+if [ -z "$(ls -A ${PATH})" ]; then
+    echo "Empty"
+    exit 1
+else
+    echo "Not Empty"
+
+    if [ $INPUT_CACHE ]; then
```

Comment:
Can we use the `[ $INPUT_CACHE ]` condition to change the header string instead of doing everything twice?
If the value is true, use `public, max-age=3600` (the default), if not use `no-store`